cypress/integration/dashboardPage.spec.js

```diff
@@ -1,5 +1,3 @@
-  /// <reference types="Cypress" />
-
   context ('DashboardPage', () => {
     beforeEach(() => {
       cy.visit('http://localhost:3000')
```

Comment:
You could add ' "baseUrl": "http://localhost:3000/" ' to your cypress.json and then it is simply ' cy.visit('/') ' in your dashboardPage.spec.js

Reply:
Thanks. Makes it easier for further tests.